target/linux/mpc85xx/files/arch/powerpc/boot/dts/firebox-t1x.dtsi

```diff
@@ -48,6 +48,18 @@
 			gpios = <&gpio0 5 GPIO_ACTIVE_LOW>;
 			label = "green:failover";
 		};
+
+		led_wap_orange: wap_orange {
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
```

Comment:
These should use &ath9k as the GPIO probably.

Reply:
The ath9k-led GPIO pin is unused in this device. The WAP (ie. "**W**iFi **A**ccess **P**oint") LED is driven by the SoC's GPIO 12 and 13. I haven't found an easy way to trigger it, though, since it's a dual color LED: green=5GHz and orange=2GHz. I thought about, maybe a hotplug script or something. For now, I left the LED without default trigger.

Reply:
I could add a `linux,default-trigger = "phy0tpt";` to one of the colors (green or orange), so the LED will at least have **some** function by default, although not exactly like in the stock FW. Your opinion?

Reply:
Might as well.

Reply:
Done.